gradle-plugin/build.gradle.kts

```diff
@@ -30,9 +30,5 @@ gradlePlugin {
 	}
 }
 
-tasks.withType<Jar> {
-	manifest {
-		attributes["Implementation-Title"] = "Telenor Boot Plugin"
-		attributes["Implementation-Version"] = project.version
-	}
-}
+// Disable Maven Publications
+publish.disable()
```

Comment:
This should be published on the plugin portal. However, this might have to change later for SNAPSHOT releases.